lib/omnibus/compressors/dmg.rb

```diff
@@ -133,64 +107,13 @@ def clean_disks
       end
     end
 
-    #
-    # Create a writable dmg we can put assets on.
-    #
-    def create_writable_dmg
-      log.info(log_key) { "Creating writable dmg" }
-
-      shellout! <<-EOH.gsub(/^ {8}/, "")
-        hdiutil create \\
-          -volname "#{volume_name}" \\
-          -fs HFS+ \\
-          -fsargs "-c c=64,a=16,e=16" \\
```

Comment:
The above `fsargs` settings are *not* lost with this change; they're indeed hardcoded in `dmgbuild` (not overridable but identical):
```py
    fs_args = "-c c=64,a=16,e=16" if filesystem != "APFS" else ""
```
- See https://github.com/dmgbuild/dmgbuild/blob/c0febffa3c4e3fb70328efe02adc289918473155/src/dmgbuild/core.py#L481 (`dmgbuild==1.6.5`)